sn_transfers/src/genesis.rs

```diff
@@ -99,7 +109,13 @@ lazy_static! {
     /// Unlike the `GENESIS_PK`, the hard coded secret_key is for testing purpose.
```

Comment:
comment needs to be updated once this PR merged.
actually, I think this block of comment can be `removed` totally.
it was a leftover from the previous refactor.
the topic is already covered by the comment of the correspondent str const